securedrop/secure_tempfile.py

```diff
@@ -61,8 +69,8 @@ def create_key(self):
         grsecurity-patched kernel it uses (for further details consult
         https://github.com/freedomofpress/securedrop/pull/477#issuecomment-168445450).
         """
-        self.key = os.urandom(self.AES_key_size / 8)
-        self.iv = os.urandom(self.AES_block_size / 8)
+        self.key = os.urandom(int(self.AES_key_size / 8))
```

Comment:
made this change because it'll bubble up a `TypeError` if `self.AES_key_size` or `self.AES_block_size` in the future is ever not divisible by 8 (since floor division doesn't necessarily return an int) which I think is useful for catching issues that future maintainers may make here